### ISSUE TITLE
access single treant from a bundle

### ISSUE THREAD

**Issue by kain88-de** [datreant/datreant.core#97]
I have an issue that I use a treant name from a Bundle `bundle['foo']` with the name `foo` being unique in the bundle. But to access the treant I still need to do `bundle['foo'][0]`. Can't I use the Treant object directly if there is only one in the bundle?

**Comment by dotsdl** [comment]
@kain88-de I understand that it's annoying to have to put a tailing `[0]` to extract the only member of the `Bundle` out of it, but I think that if sometimes you get a `Treant` and other times you get a `Bundle` from doing `bundle[treantname]` depending on duplicates this can be a real headache. 

I'm open to the suggestion, though, since this kind of behavior isn't entirely without precedent. In `pandas`, for example, extracting a row by its label with `dataframe.loc[label]` will yield that single row as a `Series` in the case the label is unique, but when there are duplicates this instead gives back a `DataFrame` of the rows with that label.

**Comment by kain88-de** [comment]
yeah I would expect a list if there are more treants with the same name and a single treant if the name is unique.

**Comment by dotsdl** [comment]
Yeah, I think this should probably happen. I know I've been annoyed with this kind of behavior in `MDAnalysis` in the past though with those magic `AtomGroup.<name>` conveniences, but there's definitely a pragmatic argument for it.

**Comment by kain88-de** [comment]
We can ask around what others think before we make the change though.
